app/views/documents/index.html.erb

```diff
@@ -68,52 +68,53 @@
               Public: <%= document.publishable? ? "Yes" : "No" %>
             </p>
         </div>
-      <div class="govuk-grid-column-one-quarter">
-        <p style="text-align: right" class="govuk-body">
-          <%= link_to "Edit", edit_planning_application_document_path(@planning_application, document) %><br />
-          <%= link_to "Archive document", planning_application_document_archive_path(document_id: document.id) if current_user.assessor? && @planning_application.in_assessment? %>
-        </p>
-      </div>
+      <% if @planning_application.can_validate? %>
+        <div class="govuk-grid-column-one-quarter">
+          <p style="text-align: right" class="govuk-body">
+            <%= link_to "Edit", edit_planning_application_document_path(@planning_application, document) %><br />
+            <%= link_to "Archive document", planning_application_document_archive_path(document_id: document.id) %>
+          </p>
+        </div>
+      <% end %>
     </li>
     <% end %>
   <% end %>
 </ul>
 
 <% unless @documents.nil? %>
```

Comment:
blank? or present? are usually safer then nil? as they cover more cases - we also seem to use them everywhere else so perhaps going with blank? here would be best